src/modelmanager.cpp

```diff
@@ -150,6 +150,19 @@ void processNodeInputs(const std::string nodeName, const rapidjson::Value::Const
     }
 }
 
+void processPipelineInputs(const rapidjson::Value::ConstMemberIterator& pipelineInputsPtr, const std::string& nodeName, std::unordered_map<std::string, std::string>& nodeOutputNameAlias, const std::string& pipelineName) {
+    for (const auto& pipelineInput : pipelineInputsPtr->value.GetArray()) {
+        const std::string pipelineInputName = pipelineInput.GetString();
+        SPDLOG_INFO("Alliasing node:{} output:{}, under alias:{}",
```

Comment:
Mapping would be better work.
While this function is called for every request, maybe DEBUG model would be more applicable here

Reply:
This is not called for every request, it is called when we read pipeline from config.json file. Most of pipeline add/reload/validate logs are INFO for development purpose, there will be one last commit before code freeze that changes log levels.

Reply:
Done